### PR TITLE
fix missing custom icons

### PR DESCRIPTION
**Pull request by Diaan**

### CODE REVIEW

File: packages/tokens/src/themes/bingel/icons.d.ts [hunk 1]
```diff
@@ -41,4 +41,19 @@ export declare const icons: {
         description: string;
         svg: string;
     };
+    cirlce: {
```
Comment (jpzwarte) [inline packages/tokens/src/themes/bingel/icons.d.ts:44]: typo

File: .github/workflows/clean.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ on:
 
 env:
   FONT_AWESOME_AUTH_TOKEN: ${{ secrets.FONT_AWESOME_AUTH_TOKEN }}
+  FIGMA_TOKEN: ${{ secrets.FIGMA_AUTH_TOKEN }}
```
Comment (jpzwarte) [inline .github/workflows/clean.yml:9]: This line is unnecessary for the `clean` workflow?

**Review by jpzwarte — APPROVED** [review]

File: packages/tokens/src/themes/bingel/icons.d.ts (hunk 1 above)
Comment (Diaan) [reply packages/tokens/src/themes/bingel/icons.d.ts:44]: image
It's like that in figma :D

File: .github/workflows/clean.yml (hunk 2 above)
Comment (Diaan) [reply .github/workflows/clean.yml:9]: for font awesome as well then?

Comment (jpzwarte) [reply .github/workflows/clean.yml:9]: Yeah, good point.

Comment (jpzwarte) [reply .github/workflows/clean.yml:9]: Or we need that token for `yarn install` to work?

File: packages/tokens/src/themes/bingel/icons.d.ts (hunk 1 above)
Comment (Diaan) [reply packages/tokens/src/themes/bingel/icons.d.ts:44]: asked the designers to fix this :D

**Pull request merged**